### PR TITLE
Add missing explain formats

### PR DESCRIPTION
**Pull request by yusra1007**
### What is changed, added or deleted? (Required)

translate doc en to cn#15601
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15601
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-explain.md [hunk 1]
```diff
@@ -192,10 +192,12 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 | FORMAT      | 作用                                                                |
 |-------------|-------------------------------------------------------------------|
 | 未指定         | 未指定 `FORMAT` 时，默认输出格式为 `row`                                                             |
-| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `brief`     | `EXPLAIN` 语句输出结果中的算子 ID 将被简化，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化      |
 | `dot`       | `EXPLAIN` 语句将输出 DOT 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
+| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `tidb_json` | `EXPLAIN` 语句将输出 JSON 格式的执行计划，算子信息存放在一个 JSON 数组中           |
+| `verbose`   | `EXPLAIN`语句将以`row`格式输出结果，并在结果中增加一列`estCost`，表示查询的估计成本。有关如何使用这种格式的更多信息，可参阅 [SQL 计划管理](/sql-plan-management.md)。|
```
Comment (qiancai) [inline sql-statements/sql-statement-explain.md:199]:
```suggestion
| `verbose`   | `EXPLAIN` 语句将以 `row` 格式输出结果，并在结果中额外包含一个 `estCost` 列，表示查询的代价估算。关于如何使用这种格式的更多信息，可参阅[执行计划管理 (SPM)](/sql-plan-management.md)。|
```

Comment (qiancai) [reply sql-statements/sql-statement-explain.md:199]: - estCost 一般表示代价估算
- 中文里 sql-plan-management.md 对应 https://docs.pingcap.com/zh/tidb/dev/sql-plan-management/ 文档

File: sql-statements/sql-statement-explain.md [hunk 2]
```diff
@@ -192,10 +192,12 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
+| `plan_cache` | `EXPLAIN` 语句将以 `row` 格式输出结果，并显示警告信息 [Plan Cache]（/sql-non-prepared-plan-cache.md#diagnostics）。
```
Comment (qiancai) [inline sql-statements/sql-statement-explain.md:200]:
```suggestion
| `plan_cache` | `EXPLAIN` 语句将以 `row` 格式输出结果，[执行计划缓存](/sql-non-prepared-plan-cache.md#诊断) 信息将作为 warning 显示。
```

Comment (qiancai) [reply sql-statements/sql-statement-explain.md:200]: 中文里 /sql-non-prepared-plan-cache.md#diagnostics 对应 https://docs.pingcap.com/zh/tidb/dev/sql-non-prepared-plan-cache#诊断

**Comment by qiancai** [comment]
Hi @yusra1007, thanks for your contribution. I've added a few comments. PTAL.

File: sql-statements/sql-statement-explain.md [hunk 3]
```diff
@@ -192,10 +192,12 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 | FORMAT      | 作用                                                                |
 |-------------|-------------------------------------------------------------------|
 | 未指定         | 未指定 `FORMAT` 时，默认输出格式为 `row`                                                             |
-| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `brief`     | `EXPLAIN` 语句输出结果中的算子 ID 将被简化，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化      |
 | `dot`       | `EXPLAIN` 语句将输出 DOT 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
+| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `tidb_json` | `EXPLAIN` 语句将输出 JSON 格式的执行计划，算子信息存放在一个 JSON 数组中           |
+| `verbose`   | `EXPLAIN` 语句将以 `row` 格式输出结果，并在结果中额外包含一个 `estCost` 列，表示查询的代价估算。有于如何使用这种格式的更多信息，可参阅[执行计划管理 (SPM)](/sql-plan-management.md)。|
+| `plan_cache` | `EXPLAIN` 语句将以 `row` 格式输出结果，[执行计划缓存](/sql-non-prepared-plan-cache.md#诊断) 信息将作为 warning 显示。
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain.md:200]:
```suggestion
| `plan_cache` | `EXPLAIN` 语句将以 `row` 格式输出结果，[执行计划缓存](/sql-non-prepared-plan-cache.md#诊断)信息将作为 warning 显示。
```

File: sql-statements/sql-statement-explain.md [hunk 4]
```diff
@@ -192,10 +192,12 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 | FORMAT      | 作用                                                                |
 |-------------|-------------------------------------------------------------------|
 | 未指定         | 未指定 `FORMAT` 时，默认输出格式为 `row`                                                             |
-| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `brief`     | `EXPLAIN` 语句输出结果中的算子 ID 将被简化，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化      |
 | `dot`       | `EXPLAIN` 语句将输出 DOT 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
+| `row`       | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
 | `tidb_json` | `EXPLAIN` 语句将输出 JSON 格式的执行计划，算子信息存放在一个 JSON 数组中           |
+| `verbose`   | `EXPLAIN` 语句将以 `row` 格式输出结果，并在结果中额外包含一个 `estCost` 列，表示查询的代价估算。有于如何使用这种格式的更多信息，可参阅[执行计划管理 (SPM)](/sql-plan-management.md)。|
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain.md:199]:
```suggestion
| `verbose`   | `EXPLAIN` 语句将以 `row` 格式输出结果，并在结果中额外包含一个 `estCost` 列，表示查询的代价估算。关于如何使用这种格式的更多信息，请参阅[执行计划管理 (SPM)](/sql-plan-management.md)。|
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**